test/e2e/framework/framework.go

```diff
@@ -443,7 +443,7 @@ func (f *Framework) WriteFileViaContainer(podName, containerName string, path st
 			return fmt.Errorf("Unsupported character in string to write: %v", c)
 		}
 	}
-	command := fmt.Sprintf("echo '%s' > '%s'", contents, path)
+	command := fmt.Sprintf("echo '%s' > '%s';sync;sync", contents, path)
```

Comment:
the legend of sync https://utcc.utoronto.ca/~cks/space/blog/unix/TheLegendOfSync